object-cache.php

```diff
@@ -508,64 +554,171 @@ function switch_to_blog( $blog_id ) {
 		$this->blog_prefix = ( is_multisite() ? $blog_id : $table_prefix );
 	}
 
-	function colorize_debug_line( $line ) {
+	function colorize_debug_line( $line, $trailing_html = '' ) {
 		$colors = array(
 			'get' => 'green',
-			'get_multi' => 'green',
+			'get_local' => 'lightgreen',
+			'get_multi' => 'fuchsia',
 			'set' => 'purple',
+			'set_local' => 'orchid',
 			'add' => 'blue',
 			'delete' => 'red',
+			'delete_local' => 'tomato',
+			'slow-ops' => 'crimson',
 		);
 
 		$cmd = substr( $line, 0, strpos( $line, ' ' ) );
 
 		// Start off with a neutral default color...
 		$color_for_cmd = 'brown';
-
 		// And if the cmd has a specific color, use that instead
 		if ( isset( $colors[ $cmd ] ) ) {
 			$color_for_cmd = $colors[ $cmd ];
 		}
 
-		$cmd2 = '<span style="color:' . esc_attr( $color_for_cmd ) . '">' . esc_html( $cmd ) . '</span>';
+		$cmd2 = "<span style='color:{$color_for_cmd}; font-weight: bold;'>" . esc_html( $cmd ) . "</span>";
 
-		return $cmd2 . esc_html( substr( $line, strlen( $cmd ) ) ) . "\n";
+		return $cmd2 . esc_html( substr( $line, strlen( $cmd ) ) ) . "$trailing_html\n";
 	}
 
-	function stats() {
-		if ( $this->stats_callback && is_callable( $this->stats_callback ) ) {
-			return call_user_func( $this->stats_callback );
+	function human_filesize( $bytes ) {
+		$sz = 'BKMGTP';
+		$factor = floor( ( strlen( $bytes ) - 1 ) / 3 );
+		$scaled = $bytes / pow( 1024, $factor );
+		$decimals = 3 - strlen( intval( $scaled ) );
+		$format = ( $decimals && intval( $scaled ) != $scaled ) ? "%.{$decimals}f" : '%d';
+
+		return sprintf( $format, $scaled ) . @$sz[ $factor ];
+	}
+
+	function js_toggle() {
+		echo "
+		<script>
+		function memcachedToggleVisibility( id, hidePrefix ) {
+			var element = document.getElementById( id );
+			if ( ! element ) {
+				return;
+			}
+
+			// Hide all element with `idPrefix` if given. Used to display only one element at a time.
+			if ( hidePrefix ) {
+				var groupStats = document.querySelectorAll( '[id^=\"' + hidePrefix + '\"]' );
+				Array.prototype.forEach.call(
```

Comment:
Could this be rewritten as `groupStats.forEach( ... )` ?

Reply:
I believe so. :)